.goreleaser.yml

```diff
@@ -19,40 +14,38 @@ builds:
 - &pulumibin
   id: pulumi
   binary: pulumi
+  dir: pkg
```

Comment:
Suggest maybe dropping these sections, if we are handling this elsewhere outside goreleaser now:

- brews (opens a pr in pulumi/homebrew-tap)
- release (opens GH rel + discussions)
- blobs (uploads to S3)

Reply:
Good call, and on that note I need to add the GH discussion to a post-release action.